client/reducers/initialState.js

```diff
@@ -1,7 +1,10 @@
 export default {
   activeUser: {},
   event: {},
-  events: [],
+  events: {
+    events: [],
```

Comment:
Do you mind changing this to something more appropriate ?
Like eventList or allEvents